docs/examples/contracts/csaf/contract.yaml

```diff
@@ -0,0 +1,16 @@
+schemaVersion: v1
+runner:
+  type: GITHUB_ACTION
```

Comment:
you can remove the runner, or is there any reason you want it to be here? Note that the runner can be auto-detected.

Reply:
No specific reason but since it's meant to be generic, let's remove to now give the impression it's tied up to that specific runner.